test/suites/integration/ami_test.go

```diff
@@ -314,3 +355,11 @@ func getInstanceAttribute(nodeName string, attribute string) *ec2.DescribeInstan
 	Expect(err).ToNot(HaveOccurred())
 	return instanceAttribute
 }
+func EventuallyExpectAMIsToExist(provider *v1alpha1.AWSNodeTemplate) {
+	Eventually(func(g Gomega) {
+		var ant v1alpha1.AWSNodeTemplate
+		g.Expect(env.Client.Get(env, client.ObjectKeyFromObject(provider), &ant)).To(Succeed())
+		g.Expect(ant.Status.AMIs).ToNot(BeNil())
+		g.Expect(len(ant.Status.AMIs)).To(BeNumerically("<", 10))
```

Comment:
Any reason you want to assert this length condition? Can this be asserted in the individual test case once this exits